articles/architecture-scenarios/b2c/authentication.md

```diff
@@ -0,0 +1,120 @@
+3. Set up your application to [receive and handle the response](#application-integration) from the Authorization Server.
+
+::: note
+If you have more than one application, the best practice is to redirect to a centralized authorization server to authenticate the user (at Auth0, this means taking advantage of Universal Login). This comes with many security and user experience benefits, including SSO. See here for more information.
+:::
+
+## Username Password Authentication
+
+Nearly every B2C application provides that ability for their customers to create a new set of credentials for this company. This is a common form of authentication that all users are familiar with.
+
+Username Password Authentication can come in multiple flavors at Auth0. If your application is a greenfield application with no existing user base, then a simple Auth0 OOTB database connection will give you everything you need to start authenticating your users. However if you have a legacy user store (such as your own database of users or an existing LDAP system) you have a couple of different options for migrating your users, see here for more information.
+
+However you end up provisioning the users for your database connection, the authentication of those users is quite similar. It requires you to present the users with a form to enter their username and password. As mentioned in the universal login section, the simplest and safest way to authenticate users with a username and password is to redirect them to a centralized login page and collect their username and password there. This allows your authorization server to determine whether they have already authenticated and skip the login form entirely when it is not needed. See here for more information.
+
+::: note
+Collecting credentials only at the centralized login page will reduce the surface area for a potential leak of user secrets. It will also reduce the need to collect credentials unnecessarily.
+:::
+
+## Anomaly Detection
+
+The reason that authentication systems are important is to prevent bad actors from accessing applications and user data that they should not. We want to place as many barriers in between those bad actors and access to our system as possible. One of the easiest ways to do this is to ensure that your anomaly detection with Auth0 is configured correctly. Take a moment to [read the docs on anomaly detection](/anomaly-detection) and ensure it is working correctly for you behind the scenes.
+
+::: note
+Anomaly Detection is handled by Auth0 and provides a great security feature for your product. If you are going to utilize it, ensure that you have set up your SMTP provider and configured your email templates before turning on email delivery to your users.
+:::
+
+## Application Integration
+
+Once you have figured out how you want to authenticate your users, the next step is to determine how and when you will initiate that authentication. Each application may have its own starting point.
```

Comment:
@PeterGFernandez I added some more color here as I think it is important to state that each application should be talking to the authentication server directly instead of relying on another application to tell it about the user.